### PR TITLE
added a little fix for the edit user page not loading

### PR DESCRIPTION
**Pull request by AdamFipke**
# Description

Fixes the NaN error when editing users through the organization page (it turns out to be a race condition of some kind that only seems to happen if the page loads too slowly).

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] This requires a run of `yarn install`

# How Has This Been Tested?

Manual testing. I just tried clicking edit user and now it will work. It will now flash for an error page for half a second before showing the proper page if it's too slow instead of returning the user to the organization page.

# Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code where needed 
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/app/pages/organization/user/[uid]/edit.tsx [hunk 1]
```diff
@@ -42,6 +42,9 @@ export default function Edit(): ReactElement {
   ) {
     return <DefaultErrorPage statusCode={404} />
   }
+  if (!uid) {
```
Comment (d3li0n) [inline packages/app/pages/organization/user/[uid]/edit.tsx:45]: This wouldn't necessarily work. Page will return 400 if data is still loading


You should add a check here:
```
return profile &&
    profile.organization.organizationRole == OrganizationRole.ADMIN &&
    organization && uid ?
```

**Review by d3li0n — CHANGES_REQUESTED** [review]
See comment

File: packages/app/pages/organization/user/[uid]/edit.tsx [hunk 2]
```diff
@@ -75,29 +75,37 @@ export default function Edit(): ReactElement {
...
+              key: course.id,
+            }
+          })
+          return userInfo
+        }),
     )
 
     if (error) {
-      router.push('/organization/settings')
+      console.log('Error: ', error)
```
Comment (d3li0n) [inline packages/app/pages/organization/user/[uid]/edit.tsx:91]: Remove console.log?

**Review by d3li0n — COMMENTED** [review]
One small comment

File: packages/app/pages/organization/user/[uid]/edit.tsx (hunk 2 above)
Comment (AdamFipke) [reply packages/app/pages/organization/user/[uid]/edit.tsx:91]: should be fine? according to lighthouse, it's best practice to log errors in the console.

Comment (d3li0n) [reply packages/app/pages/organization/user/[uid]/edit.tsx:91]: It's a bad practice to leave debug statements in productions

Comment (AdamFipke) [inline packages/app/pages/organization/user/[uid]/edit.tsx:91]:
```suggestion
```

**Comment by AdamFipke** [comment]
I resolved the comment. I think i'm just gonna merge it

**Pull request merged**